charters/rust-core/README.md

```diff
@@ -0,0 +1,277 @@
+# Rust Core Working Group Charter <!-- omit in toc -->
+
+**Start Date**: June 6th 2024
+
+**End Date**: Unknown
+
+**DRI**: Kendall Weihe (@KendallWeihe)
+
+- [Motivation and Background](#motivation-and-background)
+- [Scope](#scope)
+- [Deliverables](#deliverables)
+  - [Web5](#Web5)
+    - [\[Deliverable 1\]: Web5 API Design (APID)](#deliverable-1-web5-api-design-apid)
+    - [\[Deliverable 2\]: Web5 Rust Implementation](#deliverable-2-web5-rust-implementation)
+    - [\[Deliverable 3\]: Web5 UniFFI Binding](#deliverable-3-web5-uniffi-binding)
+    - [\[Deliverable 4\]: Web5 Kotlin Implementation](#deliverable-4-web5-kotlin-implementation)
```

Comment:
i dunno if these are sequential but i'd vote for web5 X implementation and tbdex X implementation to go hand in hand where X is language of choice

which means you'd want to do web5 API design and tbdex API design together

the reason for this is that tbdex heavily relies on web5 api to do web5 primitive work